apps/backend/.env.example

```
@@ -0,0 +1,3 @@
+DATABASE_URL=""
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)